src/seckrit.py

```diff
@@ -10,11 +10,22 @@
 
 
 def main(args):
+
     # Load the YAML file that tells us which secrets we want to load (and what to do with them).
-    manifest = load_manifest(args.manifest_file)
+    if args.manifest:
+        manifest = load_manifest(args.manifest)
+    elif "SECKRIT_MANIFEST" in os.environ:
+        manifest = yaml.safe_load(os.environ.get('SECKRIT_MANIFEST'))
+    else:
+        raise RuntimeError("Should pass --manifest argument or define SECKRIT_MANIFEST environment variable")
 
     # Create the GCP Secret Manager client using the default credential provider chain.
-    client = secretmanager_v1beta1.SecretManagerServiceClient()
+    credentials_json = json.load(os.environ.get("GOOGLE_APPLICATION_CREDENTIALS_JSON"))
```

Comment:
Need to update the comment above this message, as we are no longer exclusively using the default credential provider chain (`GOOGLE_APPLICATION_CREDENTIALS`).

What we are doing now is checking for credentials in the environment, and falling back to the default credential provider chain if not provided.